easybuild/easyconfigs/a/ASAP/ASAP-2.1-cmath.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/annotation/ImageScopeRepository.cpp b/annotation/ImageScopeRepository.cpp
```

Comment:
Please put a short comment at the top of each patch, indicating what is fixed (e.g. reference to a github issue, or the error message you were getting before implementing the patch)